src/pages/home.js

```diff
@@ -1,32 +1,69 @@
 import styled from 'styled-components';
-import BackgroundImage from '../images/Background.jpg';
 import Statistics from '../components/Statistics/Statistics';
+import CreateDiver from '../components/FormCreateDiver/CreateDiver';
+import { useState } from 'react';
+import { useLocalStorage } from 'usehooks-ts';
+import DiverInfo from '../components/DiverInfomation/DiverInfo';
 
 export default function Home({ diveData }) {
+  const [createDiverPageActive, setCreateDiverPageActive] = useState(false);
+  const [diverInfo, setDiverInfo] = useLocalStorage('diverInfo', []);
+
   return (
     <Wrapper>
-      <Headline>logbuddy</Headline>
-      <Statistics diveData={diveData} />
+      {diverInfo[0]
+        ? diverInfo?.map(
+            ({ name, organization, date, certification, cert_nr, _id }) => (
+              <DiverInfo
+                key={_id}
+                name={name}
+                organization={organization}
+                date={date}
+                certification={certification}
+                certnr={cert_nr}
+              />
+            )
+          )
+        : !createDiverPageActive && (
+            <Header onClick={handleCreateDiverPage}>create cew diver +</Header>
+          )}
+      {createDiverPageActive && (
+        <CreateDiver
+          onClickBack={handleCreateDiverPage}
+          onCreate={handleCreateDiver}
+        />
+      )}
+      {!createDiverPageActive && <Statistics diveData={diveData} />}
     </Wrapper>
   );
+
+  function handleCreateDiver(newdiverInfo) {
+    setDiverInfo([newdiverInfo]);
+    setCreateDiverPageActive(!createDiverPageActive);
+  }
```

Comment:
This function indicates for me that you only want to save one object in `diverInfo`.